fvm/evm/stdlib/contract.cdc

```diff
@@ -22,28 +22,47 @@ contract EVM {
             let balance = InternalEVM.balance(
                 address: self.bytes
             )
-
-            return Balance(flow: balance)
+            return Balance(attoflow: balance)
         }
     }
 
     access(all)
     struct Balance {
 
-        /// The balance in FLOW
+        /// The balance in atto-FLOW
+        /// Atto-FLOW is the smallest denomination of FLOW (1e18 FLOW)
+        /// that is used to store account balances inside EVM 
+        /// similar to the way WEI is used to store ETH divisible to 18 decimal places.
+        access(all)
+        var attoflow: UInt
```

Comment:
Just checking if you intentionally name it `atttoflow` not `attoFLOW` because it's a new name for metric?